src/RuleSet.php

```diff
@@ -157,9 +168,11 @@
             'modernize_types_casting' => true,
             'no_alias_functions' => true,
             'no_homoglyph_names' => true,
-            'non_printable_character' => true,
+            'non_printable_character' => [
+                // @TODO: on 3.0 switch whole rule to `=> true`, currently we use custom config that will be default on 3.0
+                'use_escape_sequences_in_strings' => true,
```

Comment:
Sf is following this new feature, which is disabled by default on 2.x and will be enabled by default on 3.x